test/e2e/run_volume_test.go

```diff
@@ -116,6 +116,11 @@ var _ = Describe("Podman run with volumes", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.OutputToString()).To(ContainSubstring("/run/test rw,nosuid,nodev,noexec,relatime - tmpfs"))
+
+		session = podmanTest.Podman([]string{"run", "--rm", "--mount", "type=tmpfs,target=/proc/sys", ALPINE, "ls", "/proc/sys"})
```

Comment:
Can we get new tests for tmpcopyup and notmpcopyup?

Reply:
yes